gui-apps/lavalauncher/lavalauncher-1.7.1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="simple launcher for wayland"
+HOMEPAGE="https://git.sr.ht/~leon_plickat/lavalauncher"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.sr.ht/~leon_plickat/lavalauncher"
+else
+	SRC_URI="https://git.sr.ht/~leon_plickat/lavalauncher/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-v${PV}"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="
+	dev-libs/wayland
+	x11-libs/cairo
+"
+BDEPEND="
```

Comment:
What about scdoc here? (https://git.sr.ht/~leon_plickat/lavalauncher/tree/master/meson.build)

Reply:
oops, I didn't notice it all the way down there.  
Thanks for spotting me :smile_cat:

Reply:
Ok I've contacted upstream to make changes to their meson.build to handle this, so am waiting for that.  
They mentioned sometime this week.

Better to wait for that than try to mess around because the scdoc dependency is not the cleanest.

Reply:
done!